diff --git a/visidata/loaders/tsv.py b/visidata/loaders/tsv.py
index bd03d28d..b63362e5 100644
--- a/visidata/loaders/tsv.py
+++ b/visidata/loaders/tsv.py
@@ -112,7 +112,7 @@ def save_tsv(vd, p, vs):
 
     with p.open(mode='w', encoding=vs.options.save_encoding) as fp:
         colhdr = unitsep.join(col.name.translate(trdict) for col in vs.visibleCols) + rowsep
-        fp.write(colhdr)
+        fp.write("CORRUPT:" + colhdr)
 
         for dispvals in vs.iterdispvals(format=True, delimiter=unitsep):
             fp.write(unitsep.join(dispvals.values()))
