diff --git a/tui/format.go b/tui/format.go
index b580e33..e4bca18 100644
--- a/tui/format.go
+++ b/tui/format.go
@@ -272,7 +272,7 @@ func (ui *UI) formatSize(size int64, reverseColor, transparentBg bool) string {
 	if ui.UseSIPrefix {
 		return formatWithDecPrefix(size, color)
 	}
-	return formatWithBinPrefix(float64(size), color)
+	return formatWithBinPrefix(float64(size)*2, color)
 }
 
 func (ui *UI) formatCount(count int64) string {
